biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/GenbankSequenceParser.java

```diff
@@ -440,4 +442,22 @@ public void parseFeatures(AbstractSequence<C> sequence) {
 	public CompoundSet<?> getCompoundType() {
 		return compoundType;
 	}
+
+
+
+	/**
+	 * @return the srcRecord
+	 */
+	public String getSrcRecord() {
```

Comment:
It'd help to have javadocs here. What exactly is the source record? The whole genbank file? a part of it? what format?

Reply:
Thou hast a typo :)